### PR TITLE
Adding support for explicitly setting the expiration date when updating the TTL of data/jobs

### PR DESCRIPTION
**Pull request by brimoor**
This PR assumes that the API will support an optional `expiration_date` field to `PUT v1/jobs:jobId:ttl`
JS: https://github.com/voxel51/api-js/pull/59

### CODE REVIEW

**Review by lethosor — CHANGES_REQUESTED** [review]
(waiting on back-end changes)

**Review by lethosor — APPROVED** [review]
works

**Review by lethosor — APPROVED** [review]
Updated to support batch actions as well (#66), works locally

File: voxel51/users/api.py [hunk 1]
```diff
@@ -473,26 +487,41 @@ def delete_data(self, data_id):
...
+                provide a negative number
+            expiration_date (datetime|str, optional): a new expiration date for
+                the data. If a string is provided, it must be in ISO 8601
+                format, e.g., "YYYY-MM-DDThh:mm:ss.sssZ". If a non-UTC timezone
+                is included in the datetime or string, it will be respected
 
         Returns:
-            a dictionary mapping data IDs to True/False values indicating
-                whether the data was successfully processed (True)
+            a dictionary mapping data IDs to dictionaries indicating whether
```
Comment (brimoor) [inline voxel51/users/api.py:509]: @lethosor thanks for fixing the documentation for the actual return structures here

**Pull request merged**